presto-main/src/test/java/com/facebook/presto/operator/TestScanFilterAndProjectOperator.java

```diff
@@ -89,6 +96,42 @@ public void testPageSource()
     }
 
     @Test
+    public void testPageSourceLazyLoad()
+            throws Exception
+    {
+        Block inputBlock = BlockAssertions.createLongsBlock(0, 10_000);
+        // If column 1 is loaded, test will fail
+        Page input = new Page(inputBlock, new LazyBlock(100, lazyBlock -> {
```

Comment:
This creates a page with mismatched block position counts. Should this pass in `2` or `inputBlock.getPositionCount()` here?